handlers/settings.go

```diff
@@ -27,13 +27,13 @@ func SettingsPollOn(c echo.Context) error {
 	gaia.Cfg.Poll = true
 	err = pipeline.StartPoller()
 	if err != nil {
-		c.String(http.StatusBadRequest, err.Error())
+		return c.String(http.StatusBadRequest, err.Error())
```

Comment:
@michelvocks We had a couple of these. :DDDD we weren't returning errors man. :D

Reply:
🙈 oh no. Good catch!